app.py

```diff
@@ -1,33 +1,54 @@
 from flask import Flask, render_template, request, redirect, url_for
+from flask_sqlalchemy import SQLAlchemy
 
 app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///pantry.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
 
-# A list to store pantry items (for demonstration purposes)
-pantry_items = []
+class PantryItem(db.Model):
```

Comment:
Good work on the SQL however I think it would be best to separate the app.py and models.py. This way the code stays consistent and easy to manipulate